WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -937,7 +937,7 @@ local methods = {
                     self.frame:SetScript("OnClick", nil)
                 end
             end
-        else
+        elseif (self.dragging) then
```

Comment:
If I read that code correctly, data is nil only iff SetDragging is called on HideOptions(). In that case we only need to reset the button that was actually being dragged.